docs/api.asciidoc

```diff
@@ -193,16 +193,27 @@ Returns the built context.
 
 [float]
 [[rails-start]]
-=== Manually hooking into Rails
+=== Rails
 
-Start the agent and hook into Rails explicitly. This is useful if you skip requiring
+Start the agent and hook into Rails manually. This is useful if you skip requiring
 the gem and using the `Railtie`.
 
 [source,ruby]
 ----
 ElasticAPM::Rails.start(server_url: 'http://localhost:8200')
 ----
 
+[float]
+[[grape-start]]
+=== Grape
+
+Start the agent and hook into Grape.
```

Comment:
Perhaps we ought to mention _when_ this is supposed to be done? Before or after starting the app? Does it matter?

Reply:
What if you run your Grape app as part of your Rails app (possible and common)? Then you don't have to explicitly start it. What happens to the config then?

Reply:
The second comment is now addressed by Grape events adding a framework name and version to transaction context when they are initiated by Rails.

Reply:
Addressing the first comment: you would put the code to start the agent and hook into Grape in config.ru or similar as is mentioned in [this comment](https://github.com/elastic/apm-agent-ruby/pull/562/files#diff-01c1a79a5a1e1b67003a1885365f2cf5R67)
I can add that clarification about config.ru here as well.